File: 3st/Juram/programmer/뒤에서 5등 위로.js
function solution(num_list) {
    var answer = [];
    num_list.sort(function(a,b){
        return a - b;              
    });
    for(let i=5; i<num_list.length; i++){
        answer.push(num_list[i]);
    }
    return answer;
}

File: 4st/Jisoo/programmer/뒤에서 5등 위로.java
import java.util.*;
class Solution {
    public int[] solution(int[] num_list) {
        Arrays.sort(num_list);

        int[] answer = new int[num_list.length - 5];

        for (int i = 5; i < num_list.length; i++) {
            answer[i - 5] = num_list[i];
        }

        return answer;
    }
}
